src/iceberg/executor/penguins.py

```diff
@@ -0,0 +1,177 @@
+"""
+Author: Aymen Alsaadi
+License: MIT
+Copyright: 2019-2020
+"""
+
+from __future__ import print_function
+import os
+import csv
+import radical.entk as re
+
+from .executor import Executor
+from ..discovery import Discovery
+
+class Penguins(Executor):
+    '''
+    :Class Penguins:
+    This class instantiates the Penguins use case.
+    :Additional Parameters:
+    :input_path: The path to the input images
+        :ouptut_path: Path to the output images
+        :model: The model name 
+        :model_path: Path of a custom model
+        :epoch: number of epochs (300) 
+    '''
+    # pylint: disable=too-many-arguments
+    def __init__(self, name, resources, project=None, input_path=None,
+                 output_path=None, gpu_ids=None, model=None,
+                 model_path=None, epoch=None):
+
+        super(Penguins, self).__init__(name=name,
+                                    resource=resources['resource'],
+                                    queue=resources['queue'],
+                                    walltime=resources['walltime'],
+                                    cpus=resources['cpus'],
+                                    gpus=resources['gpus'],
+                                    project=project)
+        self._gpu_ids = gpu_ids
```

Comment:
@AymenFJA what is `gpu_ids`? Do you want to force to use specific GPUs? What will happen if a task is scheduled to a different GPU than the expected one?